beta.go

```diff
@@ -0,0 +1,58 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"math"
+)
+
+// Beta returns the value of the complete beta function B(a, b).
```

Comment:
Add 0,0 as a special case, just to make it clear? Where are you getting the special cases? I don't see them listed on wikipedia or wolfram